### ISSUE TITLE
PG TIMESTAMP / TIME are not documented

### ISSUE THREAD

**Issue by cleboo** [sqlalchemy/sqlalchemy#8152]
### Describe the use case

Hi!

Similar to mysql, postgresql supports limiting the precision of datetime fields to a number of fractional second digits (see https://www.postgresql.org/docs/14/datatype-datetime.html). Support for this feature seems to be missing from the sqlalchemy postgresql dialect right now (docs: v1.4.37).

➡️  add a DATETIME / TIME class to the postgresql dialect that supports setting a precision.

### Databases / Backends / Drivers targeted

database: postgresql
drivers: all

### Example Use

```python
from sqlalchemy import Table, Integer
from sqlalchemy.dialects.postgresql import DATETIME

table = Table(
    "item",
    metadata,
    Column('item_id', Integer, primary_key=True),
    Column('item_ts', DATETIME(fsp=3))
)
```

### Additional context

Thanks! 🙂

**Comment by zzzeek** [comment]
surprised that hasn't already found its way here

**Comment by CaselIT** [comment]
Since time has the same option I think it would make sense add this also for time

**Comment by DanielHall977** [comment]
I'll handle this issue

**Comment by DanielHall977** [comment]
I'll submit a PR for that.

**Comment by DanielHall977** [comment]
Here's my PR:  https://github.com/sqlalchemy/sqlalchemy/pull/8185

**Comment by zzzeek** [comment]
this is merged in de4adecae6e77c0677a70e807bae5ac8cc4c6314  4d6efd06aa87cbbe2b42df260376acf43e30c308

@cleboo  this has been available all along, use sqlalchemy.dialects.postgresql.TIMESTAMP

**Comment by cleboo** [comment]
Ahh perfect, just tested it. Works!

Thanks for the quick support!
